run_viewer_local.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash -f
```

Comment:
What's the purpose of `run_viewer_local.sh`?

Reply:
run_viewer.sh uses the hutch's released version.  I want to run the version in *this* directory for debugging.  And I want to see the error messages, not hide them in a /tmp file.

Now, I entertain the idea that *always* following the symbolic link for the hutch is a bad idea, and we should kill run_viewer.sh and rename run_viewer_local.sh.